src/commands/txCommands/push.js

```diff
@@ -20,6 +20,8 @@ const pushResource$ = poData =>
 		.zip(gitBranch$, branchResourceExists$)
 		.flatMap(([gitBranch, branchResourceExists]) => {
 			if (Object.keys(poData).length) {
+				console.log('branchResourceExists: ', branchResourceExists);
+				console.log('translation poData:', poData);
```

Comment:
the poData could be pretty big, right? is there a way to summarize the data in a useful way?